config/environments/staging.rb

```diff
@@ -82,7 +79,9 @@
     domain: 'mg.idseq.net',
   }
 
-  config.action_controller.asset_host = 'staging.idseq.net'
+  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
+  # We configure IDseq to use cloudfront CDN when available.
+  config.action_controller.asset_host = ENV['CLOUDFRONT_ENDPOINT'] || 'staging.idseq.net'
```

Comment:
Nit: Thoughts on naming this `STATIC_ASSETS_ENDPOINT` instead? Just to keep it more generic.

Reply:
I hadn't considered that but I think value of being consistent with AWS docs outweighs the value of being generic. 

FWIW, we've also been consistently using AWS terms in our env vars, such as `ENV['REDISCLOUD_URL']`.